opslevel/datasource_opslevel_service_dependencies.go

```diff
@@ -0,0 +1,168 @@
+package opslevel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+// Ensure ServiceDataSource implements DataSourceWithConfigure interface
+var _ datasource.DataSourceWithConfigure = &ServiceDataSource{}
+
+func NewServiceDependenciesDataSource() datasource.DataSource {
+	return &ServiceDependenciesDataSource{}
+}
+
+// ServiceDataSource manages a Service data source.
+type ServiceDependenciesDataSource struct {
+	CommonDataSourceClient
+}
+
+// ServiceDependenciesModel describes the data source data model.
+type ServiceDependenciesModel struct {
+	Dependents   []dependentsModel   `tfsdk:"dependents"`
+	Dependencies []dependenciesModel `tfsdk:"dependencies"`
+	Service      types.String        `tfsdk:"service"`
+}
+
+type dependentsModel struct {
```

Comment:
I considered that but kept them separate in case one changes and the other does not. It's hypothetical at this point, just sharing the thinking behind this